<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TrackingAppClass</class>
 <widget class="QMainWindow" name="TrackingAppClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>996</width>
    <height>714</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TrackingApp</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="Frame_Qt">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>10</y>
      <width>640</width>
      <height>540</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="txtConsole">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>410</y>
      <width>331</width>
      <height>221</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="txtConsoleVideo">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>550</y>
      <width>641</width>
      <height>81</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>287</width>
      <height>114</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_10">
     <item alignment="Qt::AlignHCenter">
      <widget class="QLabel" name="label_4">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Manual camera rotating </string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>axis Y (up/down):    </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>axis X (left/right):</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item>
            <widget class="QLineEdit" name="txtX"/>
           </item>
           <item>
            <widget class="QLineEdit" name="txtY"/>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>200</y>
      <width>194</width>
      <height>201</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_11">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item alignment="Qt::AlignHCenter">
        <widget class="QLabel" name="label_8">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Autonomy</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_8">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_6">
           <item>
            <widget class="QLabel" name="label_6">
             <property name="text">
              <string>Number of object to track:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="trackedObjectNumber_ledit"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_7">
           <item>
            <widget class="QLabel" name="label_7">
             <property name="text">
              <string>Class of objects to track:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="cocoIDcomboBox"/>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBoxSendUART">
       <property name="text">
        <string>Send data by UART</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>321</width>
      <height>67</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="layoutDirection">
          <enum>Qt::LeftToRight</enum>
         </property>
         <property name="text">
          <string>Start object detector</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="text">
          <string>Pauze object detector</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_9">
       <item>
        <widget class="QPushButton" name="btnInitYolo">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="text">
          <string>Set</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnCaptureVideo">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Set</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>210</y>
      <width>111</width>
      <height>131</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_13">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_12">
       <item>
        <widget class="QLabel" name="label_9">
         <property name="text">
          <string>Serial Port ID:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="serialPortComboBox"/>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="btnOpenSerialPort">
       <property name="text">
        <string>Open SerialPort</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnCloseSerialPort">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Close SerialPort</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="btnShoot">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>350</y>
      <width>111</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>FIRE!</string>
    </property>
   </widget>
   <zorder>btnShoot</zorder>
   <zorder>Frame_Qt</zorder>
   <zorder>txtConsole</zorder>
   <zorder>txtConsoleVideo</zorder>
   <zorder>layoutWidget</zorder>
   <zorder>layoutWidget</zorder>
   <zorder>layoutWidget</zorder>
   <zorder>layoutWidget</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>996</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="TrackingApp.qrc"/>
 </resources>
 <connections/>
</ui>
